dto/pom.xml

```diff
@@ -21,5 +21,10 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
Do we need this explicitly defined dependency?

Reply:
Yes, because it is used by annotation jsonproperty